pyccel/codegen/printing/ccode.py

```diff
@@ -947,23 +994,23 @@ def _print_Import(self, expr):
             source = source.name[-1]
         else:
             source = self._print(source)
+        if expr.target:
+            if source.startswith('stc/'):
+                stc_name, container_type, container_key = source.split("/")
+                container = container_type.split("_")
+                return '\n'.join((f'#ifndef _{container_type.upper()}',
+                                  f'#define _{container_type.upper()}',
+                                  f'#define i_type {container_type}',
+                                  f'#define i_key {container_key}',
+                                  f'#include "{stc_name + "/" + container[0]}.h"',
+                                  '#endif\n'))
```

Comment:
Given that you don't use the target is `if expr.target:` useful here?

Reply:
actually it's not, it's removed now 
4992779c9c11524fdb9495a0db9fc8d5e3287bfa